src/pbft/pbft.h

```diff
@@ -129,25 +135,54 @@ namespace pbft
       mem = (char*)malloc(mem_size);
       bzero(mem, mem_size);
 
-      auto exec_command = ([](
-                             Byz_req* inb,
-                             Byz_rep* outb,
-                             _Byz_buffer* non_det,
-                             int client,
-                             bool ro,
-                             Seqno n) { return 0; });
-
       pbft_network = std::make_unique<PbftEnclaveNetwork>(local_id, channels);
+      pbft_config = std::make_unique<PbftConfigCCF>();
 
-      Byz_init_replica(
+      auto used_bytes = Byz_init_replica(
         node_info,
         mem,
         mem_size,
-        exec_command,
+        pbft_config->get_exec_command(),
         0,
         0,
         pbft_network.get(),
         &message_receiver_base);
+
+      service_mem = mem + used_bytes;
+
+      static std::unique_ptr<ClientProxy<uint64_t, void>> client_proxy;
```

Comment:
`auto client_proxy = std::make_unique<ClientProxy<uint64_t, void>>(*message_receiver_base);` ?

Also, we could make use of `using` to make it clearer what `uint64_t` is.